### DISCUSSION TITLE
카카오톡 인앱브라우저와 네이버 pc버전의 갱신 시간 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by illegal-username** [kimytsc/covid-rest-vaccine-macro#76] [Q&A] [answered]
카카오 pc가 막혀서 방법 찾다보니 이런 방법도 있었네요.
잘 실행중이긴 한데 카카오톡 인앱브라우저에서 업데이트 시간 간격이 대략 1초 정도인 것 같은데 더 빠르게는 할 수 없나요?
그리고 pc로 네이버 버전 실행했을땐 업데이트 시도는 1초간격으로 올라가는데 검정글씨 업데이트 시간은 5초 간격으로 업데이트되네요. 카카오는 둘 다 1초마다 변경되던데요.
어떤 문제가 있는걸까요?

참고로 카카오가 네이버 리스트의 2배입니다.

**Comment by kimytsc** [comment, marked as answer]
> 잘 실행중이긴 한데 카카오톡 인앱브라우저에서 업데이트 시간 간격이 대략 1초 정도인 것 같은데 더 빠르게는 할 수 없나요?

업데이트 간격은 delay의 값으로 설정되며, 기본값으로 1000(ms)으로 되어 있습니다. 더 빠르게 하시려면 값을 줄여주시면 됩니다.

> 그리고 pc로 네이버 버전 실행했을땐 업데이트 시도는 1초간격으로 올라가는데 검정글씨 업데이트 시간은 5초 간격으로 업데이트되네요. 카카오는 둘 다 1초마다 변경되던데요.

네이버와 카카오 모두 회색글씨의 업데이트 시간은 갱신 요청시간을 의미합니다. delay에 따라 변하는 간격이 확인되는 부분입니다.

검정글씨의 업데이트 시간의 경우는 네이버와 카카오가 서로 다르게 표시되고 있습니다.
네이버의 경우, API에서 `vaccineLastSave`의 값으로 시간(네이버에서 잔여백신 수량 갱신 시간으로 추정)을 전달해주고 있으며, 이를 표시해주도록 해놓았습니다.
카카오의 경우, API에서 전달해주는 시간이 없어 API가 정상적으로 응답되었을때의 시간을 표시해주도록 해두었습니다.
상호간에 문제는 전혀 없습니다.

**Reply by illegal-username** [reply]
앗차... 바로 위아래에 있었네요 ㅎㅎ 답변 감사합니다!!

**Reply by illegal-username** [reply]
덕분에 방금 잡았습니다 감사합니다!!
